x-pack/plugins/observability/public/components/app/news/index.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiLink,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import './index.scss';
+import { news as newsMockData } from './mock/news.mock.data';
+
+interface News {
```

Comment:
It's currently not possible to distinguish between singular and plural "News" objects.

```suggestion
interface NewsItem {
```